common/static/common/js/karma.common.conf.js

```diff
@@ -330,7 +337,11 @@ var getBaseConfig = function (config, useRequireJs) {
         // how many browser should be started simultaneous
         concurrency: Infinity,
 
-        browserNoActivityTimeout: 50000
+        browserNoActivityTimeout: 50000,
+
+        client: {
+            captureConsole: false
```

Comment:
@andy-armstrong Okay so what this line does is it prevents all logs (errors, info, debug etc.) that are logged to browser, i.e. when e.g. you do `console.log('something')` in a test it won't appear in the console. The errors, warnings and info from Karma itself will appear in console. I don't think these log from browser is necessary for the run. We can always see the logs in browser while in dev mode.

Here's the related story https://openedx.atlassian.net/browse/TNL-4700.